runner/update_status.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+if [ "${RUNNER_CUSTOM_RBAC:-}" == "true" ]; then
+    export HTTPS_PROXY=
+    export https_proxy=
+
+    APISERVER=https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}
+    SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+    NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+    TOKEN=$(cat ${SERVICEACCOUNT}/token)
+    CACERT=${SERVICEACCOUNT}/ca.crt
+
+    curl --silent -X PATCH --cacert ${CACERT} -H "Content-Type: application/merge-patch+json" -H "Authorization: Bearer ${TOKEN}" \
+        ${APISERVER}/apis/actions.summerwind.dev/v1alpha1/namespaces/${NAMESPACE}/runners/${HOSTNAME}/status \
+        -d "{\"status\":{\"phase\": \"$1\", \"message\": \"$2\"}}"
+fi
```

Comment:
All the files in this patch are missing the final line feed required by git (and many other tools). You can see an error icon being display here on GitHub in the file view. Please add a final line feed to every file.